### PR TITLE
Update go-live-checklist.md

### PR DESCRIPTION
**Pull request by meker12**
Updated v2.3 Go live checklist to include changes introduced in v2.2

## This PR is a:

- [ ] New topic
- [ ] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

Updated v2.3 Go live checklist to include changes introduced in v2.2. Fixes issue #3388.

### CODE REVIEW

File: guides/v2.3/cloud/live/go-live-checklist.md [hunk 1]
```diff
@@ -1,49 +1,33 @@
...
-menu_title: Go live checklist
-menu_order: 10
-menu_node:
 functional_areas:
   - Cloud
   - Testing
 ---
 
-Prior to entering your ticket to go live and switching the DNS, you should complete this checklist and all tests for your deployed site/store. Deploy your `master` branch to the Production environment.
+## Pre-Launch Checklist {#checklist}
```
Comment (hguthrie) [inline guides/v2.3/cloud/live/go-live-checklist.md]: A heading right after a heading is weird.

File: guides/v2.3/cloud/live/go-live-checklist.md [hunk 2]
```diff
@@ -91,14 +75,14 @@ Test and verify Fastly caching is correctly working in Production. For detailed
 
 ## Performance testing {#performance}
 
-We recommend that you review the [Magento Performance Toolkit]({{ site.mage2300url }}setup/performance-toolkit) options as part of your pre-launch readiness process.
+We recommend that you review the [Magento Performance Toolkit]({{ site.mage2200url }}setup/performance-toolkit) options as part of your pre-launch readiness process.
```
Comment (dshevtsov) [inline guides/v2.3/cloud/live/go-live-checklist.md]: just to be sure that there is no mistake:
1) here you give a link from 2.3 documentation to 2.2 documentation 
2) the link doesn't seem to be working. It should end with `/` if there is an index.html file in the `setup/performance-toolkit` directory, or `.html` if `performance-toolkit` is an HTML file

File: guides/v2.3/cloud/live/go-live-checklist.md (hunk 1 above)
Comment (meker12) [reply guides/v2.3/cloud/live/go-live-checklist.md]: Agreed-- I removed it.

**Comment by dshevtsov** [comment]
`Bug fix or improvement` relates to PRs that change how web site work.
For fixes in content, use `Content fix`.

**Comment by dshevtsov** [comment]
When you create a PR or start working on a someone's PR, start form self-assigning to a PR and adding labels to indicate target versions and type of changes

File: guides/v2.3/cloud/live/go-live-checklist.md [hunk 3]
```diff
@@ -1,49 +1,35 @@
...
+*  Set the zone’s root resource record to address the hostname
+*  Lower the value for the Time-to-Live (TTL) to refresh DNS information to correctly point customers faster to the Production store
 
   We recommend a significantly lower TTL value when switching the DNS record. This value tells the DNS how long to cache the DNS record. When shortened, it refreshes the DNS faster. For example, changing this value from 3 days to 10 minutes. Be advised, this adds load to the web server.
 
 After checking with your registrar about where to change your DNS settings, add a CNAME record for your website that points to the Fastly service: `prod.magentocloud.map.fastly.net`. If you use multiple hostnames for your site, you must add a CNAME record for each one.
 
-{: .bs-callout .bs-callout-info }
-This does not work for an [apex domain](https://blog.cloudflare.com/zone-apex-naked-domain-root-domain-cname-supp){: target="_blank"} (also referred to as a *naked* domain). You must use a DNS provider that supports forwarding DNS queries to use an apex domain.
+{:.bs-callout .bs-callout-info}
```
Comment (dshevtsov) [inline guides/v2.3/cloud/live/go-live-checklist.md]: no need to remove spaces, they serve for better readability

File: guides/v2.3/cloud/live/go-live-checklist.md (hunk 2 above)
Comment (meker12) [reply guides/v2.3/cloud/live/go-live-checklist.md]: this link goes to a GitHub repo -- it works in the published version.  However, you are correct that it should point to the 2.3 branch of the repo on this topic.   -- Good catch.

I updated the link to point to the right branch and built locally-- it opens to the correct branch of the repo: https://github.com/magento/magento2/tree/2.3/setup/performance-toolkit

Comment (dshevtsov) [reply guides/v2.3/cloud/live/go-live-checklist.md]: sorry, just noticed my mistake about the variable

File: guides/v2.3/cloud/live/go-live-checklist.md [hunk 4]
```diff
@@ -98,7 +84,7 @@ You can also test using the following 3rd party options:
...
 * [New Relic](https://support.newrelic.com/) (provided): Helps locate processes and areas of the site causing slow performance with tracked time spent per action like transmitting data, queries, Redis, and so on.
-* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided): Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks indepth: process, method call, query, load, and so on.
+* [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided): Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks in depth: process, method call, query, load, and so on.
 * [WebPageTest](https://www.webpagetest.org/) and [Pingdom](https://www.pingdom.com/): Real-time analysis of your site pages load time with different origin locations. Pingdom may cost a fee. WebPageTest is a free tool.
 
 #### Next step:
```
Comment (dshevtsov) [inline guides/v2.3/cloud/live/go-live-checklist.md]:
```suggestion
#### Next step

```
Consider changing the heading level to `##`

Comment (hguthrie) [reply guides/v2.3/cloud/live/go-live-checklist.md]: That may be common with a lot of other topics in this particular set. It's an older set, and although it is a good suggestion, may be out of the scope of this PR.

**Comment by meker12** [comment]
run tests

**Pull request merged**